docfx-extensions/Intent.DocFx.TagExtensions/MarkdownVideoTag/MarkdownVideoInlineToken.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.DocAsCode.MarkdownLite;
+
+namespace Intent.DocFx.TagExtensions.MarkdownVideoTag
+{
+    public class MarkdownVideoInlineToken : IMarkdownToken
+    {
+        // The arrangement of these parameters actually matter
```

Comment:
Why, what happens if you re-order them here and with their usages?

Reply:
I think the reflection code that DocFx uses makes the assumption that the parameters are in that order. Anything else just doesn't work at all.